tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "include": ["packages", "types"],
+  "compilerOptions": {
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "module": "esnext",
+    "target": "es2018",
+    "moduleResolution": "node",
+    "downlevelIteration": true,
+    "esModuleInterop": true,
+    "strict": true,
+    "noUnusedLocals": true,
```

Comment:
In my experience, `noUnusedLocals` and `noUnusedParameters` are too restrictive, since they block compiling even when you actively developing/debugging. Consider replacing them with lint check - https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars.md
This way unused vars won't make it to other people's computers, but you can have them locally.